src/dotnet-install.ps1

```diff
@@ -1056,6 +1069,11 @@ function Get-AkaMSDownloadLink([string]$Channel, [string]$Quality, [bool]$Intern
                     return $null
                 }
 
+                $actualRedirectUrl = Sanitize-RedirectUrl $akaMsDownloadLink
```

Comment:
I think this actuallllly might need to go in the `elseif` below? We would only want to sanitize when we reached the end of the redirects. In practice, I don't know when we have more than one so this may be okay.

Reply:
Ah, I didn't catch that we were doing repeated following. Will adjust on both scripts.

Reply:
it _looks_ like we don't do redirection-following on the bash script? I don't see any kind of loop.